go.mod

```diff
@@ -181,6 +181,7 @@ replace (
 	github.com/andreyvit/diff => github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
 	github.com/armon/circbuf => github.com/armon/circbuf v0.0.0-20150827004946-bbbad097214e
 	github.com/armon/go-metrics => github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da
+	github.com/armon/go-proxyproto => github.com/armon/go-proxyproto v0.0.0-20200108142055-f0b8253b1507
```

Comment:
https://github.com/pires/go-proxyproto seems to be better maintained and support PROXY v2 (binary) format, any reason to stick with the armon version?

Reply:
I changed the library with pires's.